TemplateApplication/Schedule/ScheduleView.swift

```diff
@@ -46,10 +50,25 @@ struct ScheduleView: View {
                 .sheet(item: $presentedContext) { presentedContext in
                     destination(withContext: presentedContext)
                 }
+                .toolbar {
+                    if !FeatureFlags.disableFirebase || ProcessInfo.processInfo.isPreviewSimulator {
+                        Button(action: {
+                            presentingAccount = true
+                        }) {
+                            Image(systemName: "person.crop.circle")
+                        }
+                            .accessibilityLabel("ACCOUNT_TITLE")
+                    }
```

Comment:
I would suggest to make this button a reusable view so we can also add it to the mock upload tab in the same position 👍

Reply:
I would assume, we also want the button to be present in the contacts view then?

Reply:
Oh yes, every tab if possible 👍

Reply:
Every tab now displays the reusable `AccountButton` view in their navigation bar. 👍